README.md

```diff
@@ -1,17 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
Please add all unnecessary files and folders like `DS_store` to `.gitignore`.

Reply:
Hello Im not quite clear on this  please help

Reply:
done

Reply:
Thank You